src/pages/finance/paymentMethod/index.jsx

```diff
@@ -72,196 +70,139 @@ export const PaymentMethodDashboard = () => {
+        },
+        {
+            label: "Type", type: "textField", field: "type"
+        },
+        {
+            label: "Name", type: "textField", field: "name",
+        },
+        {
+            label: "Fruta", type: "textField", field: "jacare",
+        },
+    ]
+
+    const handleOptionsFilterView = () => {
+        if (availableCurrencies) {
+            const options = availableCurrencies.map((item) => {
+                return {label: item.label, value: item.value}
+            })
+            options.push({label: "All", value: ''})
+
+            return options
         }
-    };
+        return undefined
+    }
+
+    const dataHeader = {
+        title: "payment methods",
+        actionButton:
+            <ActionModalPM
+                type="create"
+                size="medium"
+                onSave={handleCreatePaymentMethod}/>
+    }
+
+    const dataFilterView = {
+        isVisible: true,
+        field: searchParams.currency,
+        onClick: (value) => {
+            searchParams.currency = value
+            setCurrentQueryParams(searchParams);
+        },
+        options: handleOptionsFilterView()
+    }
+
+    const dataSummaryCards = {
+        isVisible: true,
+        title: "Last used",
+        limit: 3,
+        children:
+            lastPaymentMethods && lastPaymentMethods.map(pm =>
+                React.createElement(SummaryCard,
+                    {
+                        key: pm.id,
+                        children: <Typography variant="h3">{pm.name}</Typography>,
+                        header: {
+                            title: pm.type,
+                            icon: renderIconCards(pm.type)
+                        },
+                        caption: pm.createdAt
+                    })
+            )
+    }
+
+    const dataList = {
+        title: "All Payment methods",
+        totalPages: 10,
+        actionButtons: <Box sx={{display: "flex", gap: 1}}>
+            <SortBy
+                sortOptions={sortOptions}
+                currentQueryParams={currentQueryParams}
+                setCurrentQueryParams={setCurrentQueryParams}
+            />
+            <Filters
+                currentQueryParams={currentQueryParams} setCurrentQueryParams={setCurrentQueryParams}
+                filterOptions={filterListOptions}
+            />
+        </Box>,
+
+        children: !paymentMethods ? undefined : paymentMethods.map((paymentMethod) => {
+            return (
+                <PaymentMethodsCard
+                    key={paymentMethod.id}
+                    paymentMethod={paymentMethod}
+                    onUpdate={handleUpdate}
+                    onDelete={handleDeletePaymentMethod}
+                />
+            )
+        })
+    }
+
     useEffect(() => {
-        getPaymentMethods();
-    }, []);
+        setTimeout(() => {
+            getPaymentMethods();
+        }, "1000")
```

Comment:
#must: we must not pass magical numbers, I suggest to extract to a variable with a descriptive name, and we need a way to edit it programmatically